src/components/postDetail/ImageModal/ImageModal.module.scss

```
@@ -0,0 +1,31 @@
+.overlay {
+  @include flexbox;
+
+  position: fixed;
+  z-index: $overlay-level;
+  inset: 0;
+
+  width: 100%;
+  height: 100vh;
+
+  background-color: $opacity-black-50;
+}
+
+.image-modal {
+  @include flexbox;
+  @include thick-scrollbar;
+
+  cursor: pointer;
+  overflow-y: auto;
+  max-width: calc(100vh - 3.2rem);
+  transition: $base-transition;
+
+  &-img {
+    width: 100%;
+    height: 100%;
+
+    &-item {
+      @include image-fit(contain);
+    }
+  }
+}
```

모달이 열렸을 경우 detail 페이지 스크롤 막는 scss 추가해 주세요~
```suggestion
.body-open {
  overflow: hidden;
}
```